flink-runtime/src/test/java/org/apache/flink/runtime/clusterframework/BootstrapToolsTest.java

```diff
@@ -533,4 +547,61 @@ public void testGetEnvironmentVariablesErroneous() {
 
 		Assert.assertEquals(0, res.size());
 	}
+
+	@Test
+	public void testWriteListConfigOptionAndReload() throws IOException {
```

Comment:
How about we combine those tests into a single one. Tests that do IO operations might be expensive.

Reply:
But on the other hand I do appreciate testing a single thing at a time. I think the classes are not very testable, especially the `GlobalConfiguration`.

Reply:
I will merge this tests to a single one.